lib/ansible/utils/module_docs_fragments/decrypt.py

```diff
@@ -26,6 +26,6 @@ class ModuleDocFragment(object):
       - This option controls the autodecryption of source files using vault.
     required: false
     type: 'bool'
-    default: 'Yes'
+    default: 'yes'
```

Comment:
It looks like you didn't rebase branch on top of master, did you?

Reply:
yes, I didn't... I directly use the bug fixing branch to start a pull request.

Reply:
so eventually you'll have to sync this branch with new code (devel) anyway